.github/mcli/mcli_pytest.py

```diff
@@ -63,7 +67,7 @@
 
     pip install --upgrade --user .[all]
 
-    export COMMON_ARGS="-v --durations=20 -m '{args.pytest_markers}' --s3_bucket mosaicml-internal-integration-testing"
+    export COMMON_ARGS="-v --durations=20 -m '{args.pytest_markers}' --s3_bucket {os.environ['S3_BUCKET']}"
```

Comment:
Can you do something like pass s3 bucket arg only if its defined in the environment? or leave it hard coded